src/layers/mlp.jl

```diff
@@ -10,7 +10,7 @@ Feedforward block used in many vision transformer-like models.
 - `dense`: Type of dense layer to use in the feedforward block.
 - `activation`: Activation function to use.
 """
-function mlpblock(planes, hidden_planes; dropout = 0., dense = Dense, activation = gelu)
+function MLPBlock(planes, hidden_planes; dropout = 0., dense = Dense, activation = gelu)
```

Comment:
This probably shouldn't get capitalized if it isn't a constructor.

Reply:
Whoops. Changed that to `mlp_block` - wanted to make it more readable